javascript/rest-fastify/package.json

```diff
@@ -3,11 +3,13 @@
   "version": "1.0.0",
   "license": "MIT",
   "scripts": {
-    "dev": "node src/index.js"
+    "dev": "node src/index.js && npm run studio",
+    "studio": "npx prisma studio"
   },
   "dependencies": {
     "@prisma/client": "2.13.1",
     "fastify": "3.9.2",
+    "fastify-blipp": "^3.0.0",
```

Comment:
I'd remove `fastify-blipp` so that the example focuses on the bare minimum to integrate Prisma with Fastify